### ISSUE TITLE
[5.4] Sequence doesnt work

### ISSUE THREAD

**Issue by brunorsantos** [yajra/laravel-oci8#254]
It seems that versions 5.4 does not work using sequences from oracle.

I'm using the same code from one 5.3 project of mine, but when I'm trying to insert a registry, the insert clause does not replace the primary key by the oracle sequence  specified on the model.

```php
class PreInscricao extends Authenticatable
{
    
    protected $table = 'dbowner.insc';
    protected $primaryKey = 'codinsc';
    public $sequence = 'dbowner.insc_s';
```

Inserting:

```php
        $preInscricao = new PreInscricao();
        $preInscricao->nomcan = $request->name; 
        $preInscricao->emailcan = $request->email;
        $preInscricao->save();
```

This is part of the error that shows that the primary key was not replaced:

Statement : insert into "DBOWNER"."INSC" ("NOMCAN", "EMAILCAN") values (:p0, :p1) returning "CODINSC" into :p2
Bindings : [Benedito Joao Beleleu,testmail@gmail.com,0]

Getting the sequence manually, it worked... Like this:

```php
        $preInscricao = new PreInscricao();
        $preInscricao->nomcan = $request->name; 
        $preInscricao->emailcan = $request->email;
        $sequence = \DB::getSequence();
        $codinsc =  $sequence->nextValue('dbowner.insc_s');
        $preInscricao->codinsc = $codinsc;   

        $preInscricao->save();
```

- Operating System
- PHP Version 7.1
- Laravel Version 5.4.10
- Laravel-OCI8 Version 5.4.2

**Comment by yajra** [comment]
Patch released on v5.4.3. Thanks for reporting! 🚀

**Comment by luiz-andrade** [comment]
Good evening.
Is it possible to correct the problem in version 5.1. *?
